resources/options_lookup.tsv

```diff
@@ -1226,6 +1246,9 @@ HVAC System Cooling	"AC, SEER 10"	ResidentialHVACCentralAirConditionerSingleSpee
 HVAC System Cooling	"AC, SEER 13"	ResidentialHVACCentralAirConditionerSingleSpeed	seer=13.0	eer=11.1	shr=0.73	fan_power_rated=0.365	fan_power_installed=0.5	crankcase_capacity=0.0	crankcase_temp=55	eer_capacity_derate_1ton=1.0	eer_capacity_derate_2ton=1.0	eer_capacity_derate_3ton=1.0	eer_capacity_derate_4ton=1.0	eer_capacity_derate_5ton=1.0	capacity=autosize	dse=NA																																		
 		ResidentialAirflow																																																
 		ResidentialHVACSizing	show_debug_info=false																																															
+HVAC System Cooling	"AC, SEER 14"	ResidentialHVACCentralAirConditionerSingleSpeed	seer=14.0	eer=12.0	shr=0.73	fan_power_rated=0.365	fan_power_installed=0.5	crankcase_capacity=0.0	crankcase_temp=55	eer_capacity_derate_1ton=1.0	eer_capacity_derate_2ton=1.0	eer_capacity_derate_3ton=1.0	eer_capacity_derate_4ton=1.0	eer_capacity_derate_5ton=1.0	capacity=autosize	dse=NA																																		

```

Comment:
I think sticking with this is fine given @shorowit 's comment, but for your reference, fed standard requires eer=12.2 in the southwest for <45,000 Btu/hr split ACs (and eer=11.7 for larger units) : http://icpindexing.toddsit.com/userfiles//AC_Map_Chart(3).jpg.

Reply:
Thank you, that is good reference knowledge.

Reply:
@shorowit @ejhw btw I created this measure, so the intent of the creator is suspect. Should I change this to `eer=12.2` to account for the southwest US, or leave it at 12 as a middle ground based on Eric's picture?

Reply:
@vtnate I say leave as is, to guarantee that it works (since it was an option in BEopt). Sometimes, for example, changing EER without changing airflow, can cause errors, though 12 to 12.2 is a very minor change.